coverage.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+echo "Executing coverage analysis script"
+# Programs below must be executed for gcov to generate .gcda files
+./src/tlog/tlog-rec-session -c whoami
+./src/tlog/tlog-rec
+./src/tlog/tlog-play
+lcov --compat-libtool --directory . --capture --output-file coverage.info
+lcov --remove coverage.info '*src/tltest*' '*lib/tltest*' '*include/tlog*' '*include/tltest*' '*usr/include/curl*' -o coverage.info
+coveralls-lcov coverage.info
```

Comment:
AFAIK, nothing except Travis CI would find this code useful and there's not much of it, so please don't add another file in the top directory and keep these lines inside .travis.yml.